apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/KafkaConsumerInterceptor.java

```diff
@@ -88,6 +91,8 @@ public Object afterMethod(EnhancedInstance objInst, Method method, Object[] allA
     @Override
     public void handleMethodException(EnhancedInstance objInst, Method method, Object[] allArguments,
         Class<?>[] argumentsTypes, Throwable t) {
-        ContextManager.activeSpan().errorOccurred().log(t);
+        if (ContextManager.isActive()) {
```

Comment:
Why we have inactive cases?

Reply:
`KafkaConsumerInterceptor` create entry span in `afterMethod`, but `handleMethodException` is called before `afterMethod`, and before the creation of entry span, we can not ensure there is an active span.